lib/console_web/controllers/data_credit_controller.ex

```diff
@@ -131,85 +146,97 @@ defmodule ConsoleWeb.DataCreditController do
   end
 
   def remove_payment_method(conn, %{ "paymentId" => paymentId, "latestAddedCardId" => latestAddedCardId }) do
```

Comment:
not needed since they wont even be able to create payment methods?

Reply:
i was restricting any POSTs to stripe, but yea, makes sense. pushed